addons/stock/models/stock_picking.py

```diff
@@ -697,7 +702,14 @@ def _button_validate(self):
             if backorder_pickings:
                 return self._generate_backorder_wizard(backorder_pickings)
 
-        return self._action_done()
+        # Zero Quantity Count Wizard
+        if self.env.user.has_group('stock.group_stock_zero_quantity_count') and not self.env.context.get('skip_zqc'):
+            empty_locations = self._get_empty_locations()
+            if empty_locations:
+                return self._generate_zqc_wizard(empty_locations)
+
+        self._action_done()
+        return
```

Comment:
before we returned the result of self._action_done(), now we don't, is there a reason?